projects/vdk-plugins/vdk-control-api-auth/src/vdk/plugin/control_api_auth/auth_exception.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+
+class VDKAuthException(Exception):
```

Comment:
The vdk-control-cli's VdkException extends `click.UsageError` in order to allow click to properly display the error message in terminal.
I did not re-use the exceptions from vdk-core/vdk-control-cli to avoid circular dependencies.

Reply:
Not exactly. There will be a few more PRs for the different authentication flows. Then I'll enable the release step of the librry, and only after that there will be a PR with the vdk-control-cli integration with this library.